src/VirtoCommerce.ExperienceApiModule.Core/Extensions/ProvideUserContextExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
Please use this code to store and get parent  object data for nested schema types 
https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Extensions/ProvideUserContextExtensions.cs

Reply:
would be better to move this code in the Core project to be able to share such logic between all xAPI projects